sitemedia/js/controllers/ittpanel_controller.js

```diff
@@ -74,50 +81,57 @@ export default class extends Controller {
+        // by its computed line height from CSS, and applying a floor function
+        return Math.floor(
+            el.getBoundingClientRect().height /
+                parseInt(getComputedStyle(el).getPropertyValue("line-height"))
+        );
+    }
+
     alignLines() {
-        // align each line of transcription to translation
+        // first, align tops of lists (using inline styles)
+        const edTop = this.transcriptionTarget
+            .querySelector("ol")
+            .getBoundingClientRect().top;
+        const trTop = this.translationTarget
+            .querySelector("ol")
+            .getBoundingClientRect().top;
+        if (edTop < trTop) {
+            this.transcriptionTarget.querySelector("ol").style.paddingTop = `${
+                trTop - edTop
+            }px`;
+        } else if (trTop < edTop) {
+            this.translationTarget.querySelector("ol").style.paddingTop = `${
+                edTop - trTop
+            }px`;
+        }
+
+        // then, align each line of transcription to translation
         const edLines = this.transcriptionTarget.querySelectorAll("li");
         const trLines = this.translationTarget.querySelectorAll("li");
         // only align as many lines as we need to
-        const [lessLines, moreLines] =
-            edLines.length < trLines.length
-                ? [trLines, edLines]
-                : [edLines, trLines];
-        lessLines.forEach((li, i) => {
-            if (li && moreLines[i]) {
-                // compare top of lines by position in list
-                const thisLiPos = li.getBoundingClientRect();
-                const thatLiPos = moreLines[i].getBoundingClientRect();
-                // add padding-top to the elements that need it
-                if (thisLiPos.top < thatLiPos.top) {
-                    li.style.paddingTop = `${thatLiPos.top - thisLiPos.top}px`;
-                } else if (thisLiPos.top > thatLiPos.top) {
-                    moreLines[i].style.paddingTop = `${
-                        thisLiPos.top - thatLiPos.top
-                    }px`;
+        const minLines = Math.min(edLines.length, trLines.length);
+        for (let i = 0; i < minLines; i++) {
+            if (edLines[i] && trLines[i]) {
+                // calculate number of lines based on line height
+                const edLineCount = this.getLineCount(edLines[i]);
+                const trLineCount = this.getLineCount(trLines[i]);
+                // set data-lines attribute on each li according to which is longer
+                if (edLineCount > trLineCount) {
+                    edLines[i].setAttribute("data-lines", edLineCount);
+                    trLines[i].setAttribute("data-lines", edLineCount);
+                } else {
+                    edLines[i].setAttribute("data-lines", trLineCount);
+                    trLines[i].setAttribute("data-lines", trLineCount);
                 }
```

Comment:
If you're always setting the data attribute on both, can't you just do this?

```suggestion
                const maxLineCount = Math.max(this.getLineCount(edLines[i]), this.getLineCount(trLines[i]));
                // set data-lines attribute on each li according to which is longer
                edLines[i].setAttribute("data-lines", maxLineCount);
                trLines[i].setAttribute("data-lines", maxLineCount);

```